ghost/i18n/locales/uk/comments.json

```diff
@@ -1,73 +1,73 @@
-    "Local resident": "",
-    "Member discussion": "",
-    "Name": "",
-    "Neurosurgeon": "",
-    "One day ago": "",
-    "One hour ago": "",
-    "One minute ago": "",
-    "One month ago": "",
-    "One week ago": "",
-    "One year ago": "",
-    "Reply": "",
-    "Reply to comment": "",
-    "Report": "",
-    "Report comment": "",
-    "Report this comment": "",
-    "Report this comment?": "",
-    "Save": "",
-    "Sending": "",
-    "Sent": "",
-    "Show": "",
-    "Show {{amount}} more replies": "",
-    "Show {{amount}} previous comments": "",
-    "Show 1 more reply": "",
-    "Show 1 previous comment": "",
-    "Show comment": "",
-    "Sign in": "",
-    "Sign up now": "",
-    "Start the conversation": "",
-    "This comment has been hidden.": "",
-    "This comment has been removed.": "",
-    "Upgrade now": "",
-    "Yesterday": "",
-    "You want to report this comment?": "",
-    "Your request will be sent to the owner of this site.": ""
+    "{{amount}} characters left": "{{amount}} символів залишилось",
+    "{{amount}} comments": "{{amount}} коментарів",
+    "{{amount}} days ago": "{{amount}} днів тому",
+    "{{amount}} hours ago": "{{amount}} годин тому",
+    "{{amount}} minutes ago": "{{amount}} хвилин тому",
+    "{{amount}} months ago": "{{amount}} місяців тому",
+    "{{amount}} more": "ще {{amount}}",
+    "{{amount}} seconds ago": "{{amount}} секунд тому",
+    "{{amount}} weeks ago": "{{amount}} тижнів тому",
+    "{{amount}} years ago": "{{amount}} років тому",
+    "1 comment": "1 коментар",
+    "Add comment": "Додати коментар",
+    "Add context to your comment, share your name and expertise to foster a healthy discussion.": "Додайте контекст до свого коментаря, поділіться своїм іменем і досвідом, щоб сприяти здоровій дискусії.",
+    "Add reply": "Додати відповідь",
+    "Already a member?": "Ви є учасником?",
+    "Anonymous": "Анонім",
+    "Become a member of {{publication}} to start commenting.": "Станьте учасником {{publication}} щоб почати коментувати.",
+    "Become a paid member of {{publication}} to start commenting.": "Станьте платним учасником {{publication}} щоб почати коментувати.",
+    "Cancel": "Відмінити",
+    "Comment": "Коментувати",
+    "Complete your profile": "Заповніть свій профіль",
+    "Delete": "Видалити",
+    "Deleted member": "Видалити учасника",
+    "Discussion": "Обговорення",
+    "Edit": "Редагувати",
+    "Edit this comment": "Редагувати цей коментар",
+    "Edited": "Відредагований",
+    "Enter your name": "Введіть своє імʼя",
+    "Expertise": "Експертність",
+    "Founder @ Acme Inc": "Засновник @ Acme Inc",
+    "Full-time parent": "Виховую дітей",
+    "Head of Marketing at Acme, Inc": "Голова продажів в Acme, Inc",
+    "Hide": "Сховати",
+    "Hide comment": "Сховати коментар",
+    "Jamie Larson": "Джеймі Ларсон (Jamie Larson)",
+    "Join the discussion": "Долучитися до обговорення",
+    "Just now": "Прямо зараз",
+    "Local resident": "Місцевий",
```

Comment:
@cathysarisky Hm, this one is tough. I don't even know what should be here. I am a little bit confused.
If we'll try to directly translate the `Local resident` it will be `Місцевий житель`, or commonly used a short form as just `Місцевий`. Taking in account you comment I assume that we talk about an expert of some topic or thread in the comments. So, then we can call it as `Місцевий експерт`, at least let's give it a try